File: CrimsonClubs/CrimsonClubs.Models/Dtos/ApiAuthDto.cs
using CrimsonClubs.Models.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CrimsonClubs.Models.Dtos
{
    public class ApiAuthDto
    {
        public string Token { get; set; }
        public UserDto User { get; set; }
        
        public ApiAuthDto()
        {

        }

        public ApiAuthDto(string token, User user)
        {
            Token = token;
            User = new UserDto(user);
        }
    }
}

File: Android/CrimsonClubs/app/src/main/java/crimsonclubs/uacs/android/crimsonclubs/ApiAuthDto.java
package crimsonclubs.uacs.android.crimsonclubs;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;


public class ApiAuthDto {

    @Expose
    @SerializedName("token")
    public String token;
    @Expose
    @SerializedName("user")
    public UserDto user;

}